app/views/consent_forms/record.html.erb

```diff
@@ -1,3 +1,7 @@
-<%= render "confirmation_agreed" %>
+<% if @consent_form.any_health_answers_truthy? %>
```

Comment:
This is really just about naming, but ... should the view really be deciding what constitutes needing triage? I think it feels like `@consent_form` should have a method `needs_triage?`, but which will essentially just be `any_health_answers_truthy?`

Reply:
Was thinking about that. We could add another method to the consent form to make this clearer.

Reply:
I think it would be ok if we just renamed the method in consent form, tbh, but either approach works for me.